build.gradle

```diff
@@ -228,3 +264,64 @@ task dependencySourcesZip(type: Zip, dependsOn: 'buildSourcePathsArray' ) {
  
     from _sourcePaths
 }
+
+asciidoctor {
+    dependsOn test
+    sources {
+        include 'api.adoc'
+    }
+    backends = ['pdf']
+    attributes = [
+        doctype: 'book',
+        toc: 'left',
+        toclevels: '2',
+        numbered: '',
+        sectlinks: '',
+        sectanchors: '',
+        hardbreaks: '',
+        generated: generatedAsciidoc
+    ]
+}
+
+bintrayUpload.dependsOn asciidoctor
+
+bintray {
+	user = project.hasProperty('bintrayUser') ? project.property('bintrayUser') : System.getenv('BINTRAY_USER')
+	key = project.hasProperty('bintrayApiKey') ? project.property('bintrayApiKey') : System.getenv('BINTRAY_API_KEY')
+    filesSpec {
+       from file(asciidoctor.outputDir.path + '/pdf/api.pdf')
+       into '.'
+    }
+    dryRun = false
+    publish = true
+    override = false
+    pkg {
+		repo = 'smi-service-dell-server-inventory'
```

Comment:
externalize this property in build.gradle

Reply:
Done